plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -615,6 +617,8 @@ def add_rpm_unit(self, metadata_files, unit):
             unit.save()
         except NotUniqueError:
             unit = unit.__class__.objects.filter(**unit.unit_key).first()
+        if rpm_parse.signature_enabled(self.config):
```

Comment:
It would be good not to raise the exception for each unit that did not pass the signature check, but instead just send a helpful message in the logs. And not to flud the logs it would be good to set some max limit and for the rest set a counter that would do the count for units failed the check. So we would have a final message '12345 RPMs failed the signature check and were not imported into the repo'
Problem is - it is hard to find a proper place where this could be implemented.